libnymea-zigbee/libnymea-zigbee.pro

```diff
@@ -3,8 +3,14 @@ include(../config.pri)
 TARGET = nymea-zigbee1
 TEMPLATE = lib
 
-CONFIG += link_pkgconfig
-PKGCONFIG += libudev
+
+disable_udev {
```

Comment:
actually, thinking about this, even easier would be
```
CONFIG += link_pkgconfig
package_exists(libudev) {
    PKGCONFIG += libudev
    DEFINES += WITH_UDEV
}
```

Reply:
my only concern: `package_exists` is not available for all qt verions, and if you don't need libudev, you don't need pkgconfig.

Reply:
packagesExist() I mean... What makes you believe it's not available?

Reply:
ah, from pkg config, right...makes sense

Reply:
fixed with last commit

Reply:
also aligns with cmake style, as the normal way to do it there is to 

`find_packages(name, VAR_IF_TRUE, VAR_IF_FALSE)` 
and then build the code around the outcome of that.